regtests/client/python/cli/polaris_cli.py

```diff
@@ -55,6 +67,27 @@ def execute(args=None):
                 PolarisCli._try_print_exception(e)
                 sys.exit(1)
 
+    @staticmethod
+    def _merge_with_config(options):
+        config = PolarisCli._load_config()
+        if not config:
+            return
+
+        for attr in ['client_id', 'client_secret', 'host', 'port']:
+            if not getattr(options, attr) and config.get(attr):
+                setattr(options, attr, config[attr])
```

Comment:
This doesn't overwrite the default values of host and port, will make the change to overwrite.